### ISSUE TITLE
Poor Handlebars syntax highlighting in Material Spacegray

### ISSUE THREAD

**Issue by sotojuan** [saadq/Materialize#61]
First of all let me say this is an issue I've had with plenty of color schemes in Sublime Text, so it may not be something wrong with this project.

Using the Sublime Handlebars extensions, syntax highlighting for `.hbs`. or `.handlebars` files is poor in a number of color schemes, specifically Spacegray. The main issue I have is that the opening and closing brackets as well as the "control" keywords (`if`, `else`, anything with a `#` before it) are all the same color. It'd be better if those at least were different. The properties passed to an Ember component (first screenshot) should also be different colors, not just the strings, but that isn't a big deal.

The problem is not with the extension by the way, it works in other color schemes. It's just Spacegray, Spaceblack, Spacegray Oceanic, etc that have this issue. Thanks.

### Theme Issue:

1. OS: OS X El Capitan
2. Sublime Text version: 3103
3. Materialize version: 2.5.11
4. Description of issue: Poor syntax highlighting for Handlebars
5. Steps to reproduce (if appropriate): Load up any Handlebars file
6. Screenshot of issue (if appropriate):
<img width="722" alt="screen shot 2016-05-07 at 7 31 55 pm" src="https://cloud.githubusercontent.com/assets/8217766/15095221/9e2439aa-148a-11e6-9b77-e27e0d576464.png">

<img width="1311" alt="screen shot 2016-05-07 at 7 37 23 pm" src="https://cloud.githubusercontent.com/assets/8217766/15095244/58adc318-148b-11e6-92d9-6fe0f13d3407.png">

**Comment by sotojuan** [comment]
Here's how it looks like on Atom Spacegray, for example:

<img width="732" alt="screen shot 2016-05-07 at 8 10 43 pm" src="https://cloud.githubusercontent.com/assets/8217766/15095367/0655ce0c-1491-11e6-9f7d-04fcccbdd7e9.png">

**Comment by saadq** [comment]
Thanks for raising the issue. Yeah, this seems to be a problem with the original base16 color schemes. I can add support for Handlebars syntax, but there are some limitations due to the Handlebars language definition (I assume you're using [this](https://github.com/daaain/Handlebars) one.) It doesn't have any syntax definitions for the properties, so I can't make that a different color. So the best I can do right now is:

<img width="1392" alt="screen shot 2016-05-07 at 8 41 55 pm" src="https://cloud.githubusercontent.com/assets/5678694/15095437/2e696a40-1494-11e6-8385-00a61e271483.png">
<img width="1392" alt="screen shot 2016-05-07 at 8 41 59 pm" src="https://cloud.githubusercontent.com/assets/5678694/15095438/2e69b1e4-1494-11e6-8cbe-091d78e85469.png">

Let me know if I misunderstood what you meant by any of those color changes, I haven't actually used Handlebars before. And I'll raise an issue on the language definition and see if he would be down for adding support for parameters. 

If you're cool with this I'll prob be able to push this by tomorrow since I had a couple of other pending issues to resolve as well.

**Comment by saadq** [comment]
Also, I can actually control the colors in a line like this:

    {{if buttonDisabled "disabled" "green white-text"}}"

In a situation like this, would it make more sense for `if` and `buttonDisabled` to be the same color or have them be different like in the screenshot above?

**Comment by sotojuan** [comment]
Yeah your screenshots look fine, thanks! I'll also post an issue in the Handlebars Sublime syntax just in case, but for now this is good enough.

**Comment by sotojuan** [comment]
Update: Would it be possible to change the orange to another color entirely? Not sure what is the best alternative, but anything can be better than that.

**Comment by saadq** [comment]
Thanks for creating the issue over there, actually got a very bad case of the flu over the weekend. I've made some color modifications and pushed the changes to Package Control so the color schemes should update within a few hours. Once your scheme has updated let me know what you think about the color changes.  If you have any suggestions I can just do a quick patch. So just let me know if there's too much blue or too much red now or w/e. I tried to make it look a bit more similar to the Atom version.

I'll leave the issue open for now.

**Comment by sotojuan** [comment]
No problem, take your time and feel better! Either way the Handlebars project doesn't seem very active so it'll probably be a while until we get a reply anyway—thought I might as well do it now. I'll report back on the new colors later. Thanks.

**Comment by sotojuan** [comment]
Just updated—looking good! Thanks.

**Comment by saadq** [comment]
Okay, I'll close this issue but I'm watching the issue you created so I'll make updates if he replies.

**Comment by Shak-eah** [comment]
Has this syntax highlighting been updated? I'm facing the exact same issue and my projects are heavily Ember-oriented. I absolutely love this theme but the same color between the {{ and the operators right after is throwing me off a lot. Currently, with material theme, my snippets looks like this:

<img width="599" alt="screen shot 2017-05-10 at 10 54 29 am" src="https://cloud.githubusercontent.com/assets/4359715/25905106/168b3280-356f-11e7-80e6-8fb73156a4c7.png">

versus default:

<img width="608" alt="screen shot 2017-05-10 at 10 56 46 am" src="https://cloud.githubusercontent.com/assets/4359715/25905233/724965ce-356f-11e7-8169-f7461f644310.png">

I saw here : https://github.com/daaain/Handlebars/issues/81 that the author of the handlebars package said he added definitions (?), but even with the latest versions, I'm not seeing the theme update with the screenshots you provided on May 7, 2016. Has something changed? This would be a tremendous help!
